workflow/react-project-page/src/components/Main/MainTitle/MainTitle.module.scss

```diff
@@ -0,0 +1,22 @@
+@import '../../../variables.scss';
+
+.mainTitle {
+  margin-bottom: 50px;
+  @include mainTitle;
+}
+
+@media screen and (max-width: 768px) {
+  .mainTitle {
+    margin-bottom: 40px;
+    @include mainTitle;
+    font-size: 4rem;
+  }
+}
+
+@media screen and (max-width: 350px) {
+  .mainTitle {
+    margin-bottom: 40px;
```

Comment:
the same line 10
`max-width: 768px` means that properties inside will be used from `0px` to `768px` (so 350px is included)